tilelang/testing/__init__.py

```
@@ -16,13 +16,37 @@
     "requires_cuda_compute_version",
     "process_func",
     "regression",
 ] + [f"requires_cuda_compute_version_{op}" for op in ("ge", "gt", "le", "lt", "eq")]
 
 
+def _get_rocm_gcn_arch() -> str:
+    try:
+        import tvm
+        target = tvm.target.Target("auto")
+        return target.attrs.get("mcpu", "")
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)